src/hotspot/share/prims/whitebox.cpp

```diff
@@ -23,8 +23,6 @@
  */
 
 #include "precompiled.hpp"
-#include <new>
```

Comment:
Why was `<new>` removed?

Reply:
We tend to pull it in via memory/allocation.hpp. I can revert this, but I'm not sure it is important. There's no consistency in when we include `<new>` or not.